mantle/cmd/kola/qemuexec.go

```diff
@@ -205,6 +208,10 @@ func runQemuExec(cmd *cobra.Command, args []string) error {
 	builder := platform.NewQemuBuilder()
 	defer builder.Close()
 
+	if architecture != "" {
```

Comment:
Should we do an upfront check that the value is one of the supported ones? Ideally `pflag` would support choice parameters like Python's argparse, but that doesn't seem to be the case.

Reply:
Or maybe a cleaner way to do this is to make `builder.Architecture` private and have a `SetArchitecture()` setter instead.